packages/localize-monetary-amount/src/localize-monetary-amount.ts

```diff
@@ -0,0 +1,980 @@
+		case 'PLN': // Polish złoty
+			return 'zł';
+		case 'PRB': // Transnistrian ruble
+			return 'p.';
+		case 'PYG': // Paraguayan guarani
+			return '₲';
+		case 'QAR': // Qatari riyal
+			return 'ر.ق';
+		case 'RON': // Romanian leu
+			return 'lei';
+		case 'RSD': // Serbian dinar
+			return 'дин.';
+		case 'RUB': // Russian ruble, also Pokédollar
+			return '₽';
+		case 'SAR': // Saudi riyal
+			return 'ر.س';
+		case 'SDG': // Sudanese pound
+			return 'ج.س.';
+		case 'SLL': // Sierra Leonean leone
+			return 'Le';
+		case 'SLS': // Somaliland shilling
+			return 'Sl';
+		case 'STN': // São Tomé and Príncipe dobra
+			return 'Db';
+		case 'SYP': // Syrian pound
+			return '£S';
+		case 'SZL': // Swazi lilangeni
+			return 'L';
+		case 'THB': // Thai baht
+			return '฿';
+		case 'TJS': // Tajikistani somoni
+			return 'SM';
+		case 'TMT': // Turkmenistan manat
+			return 'm';
+		case 'TND': // Tunisian dinar
+			return 'د.ت';
+		case 'TOP': // Tongan pa'anga
+			return 'T$';
+		case 'TRY': // Turkish lira
+			return '₺';
+		case 'UAH': // Ukrainian hryvnia
+			return '₴';
+		case 'UZS': // Uzbekistani so'm
+			return 'сўм';
+		case 'VES': // Venezuelan bolívar
+			return 'Bs.S.';
+		case 'VND': // Vietnamese đồng
+			return '₫';
+		case 'VUV': // Vanuatu vatu
+			return 'Vt';
+		case 'WST': // Samoan tālā
+			return 'T';
+		case 'XPF': // CFP franc
+			return '₣';
+		case 'YER': // Yemeni rial
+			return '﷼';
+		case 'ZAR': // South African rand
+			return 'R';
+		case 'ZMW': // Zambian kwacha
+			return 'ZK';
+
+		default:
+			throw `Symbol for currency code not defined: ${ currencyCode }`;
+	}
+}
+
+/**
+ * Type representing a digit separator custom.
+ *
+ *   * `decimalSeparator` is the symbol used for the radix point
+ *   * `groupSeparator` is the symbol used to demarcate digit groups
+ *
+ * @see separatorsForLocale()
+ */
+interface DigitSeparators {
+	decimalSeparator: string;
+	groupSeparator: string;
+}
+
+/**
+ * Customary radix point and group separators in a given locale. Note
+ * that we're using non-breaking spaces.
+ *
+ * @param {string} localeCode
+ *   ISO 631-1 language code with an optional ISO 3166-1 alpha-2 region code,
+ *   separated by a hyphen. Examples: 'en', 'en-gb', 'fr-be'.
+ * @returns {DigitSeparators}
+ *   Decimal and group separators
+ */
+function separatorsForLocale( localeCode: LocaleCode< Normalized > ): DigitSeparators {
+	switch ( localeCode ) {
+		// 10,000.00
+		case 'en-hk': // English - Hong Kong
+		case 'en-ie': // English - Ireland
+		case 'en-gb': // English - United Kingdom
+		case 'en-nz': // English - New Zealand
+		case 'en-us': // English - United States
+		case 'es-mx': // Spanish - Mexico
+		case 'hi': // Hindi
+		case 'he': // Hebrew
+		case 'ja': // Japanese
+		case 'ko': // Korean
+			return { decimalSeparator: '.', groupSeparator: ',' };
+
+		// 10 000.00
+		case 'en-au': // English - Australia
+		case 'en-ca': // Canada
+			return { decimalSeparator: '.', groupSeparator: '\u00A0' };
```

Comment:
With `"\u00A0"`, do you think it's worth extracting to a constant `NO_BREAK_SPACE` or `NBSP`?

Reply:
Good idea - this makes the intent more clear.